### PR TITLE
Added file

### PR DESCRIPTION
**Pull request by Poojap1512**
"Please review this, @casey-collab".

### CODE REVIEW

File: english/cry-out-for-mommy/oh_daddy.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Plz daddy!! SAVE ME!!! Help papa!!!
```
Comment (casey-collab) [inline english/cry-out-for-mommy/oh_daddy.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/cry-out-for-mommy/oh_daddy.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Poojap1512** [comment]
"Please review this, @casey-collab".

File: english/cry-out-for-mommy/oh_daddy.md (hunk 1 above)
Comment (casey-collab) [inline english/cry-out-for-mommy/oh_daddy.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/cry-out-for-daddy/oh_daddy.md [hunk 2]
```diff
@@ -0,0 +1 @@
+oh dad!!Save me!!
```
Comment (casey-collab) [inline english/cry-out-for-daddy/oh_daddy.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

File: english/cry-out-for-mommy/oh_daddy.md (hunk 1 above)
Comment (casey-collab) [inline english/cry-out-for-mommy/oh_daddy.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Poojap1512** [comment]
"Please review this, @casey-collab".

File: english/cry-out-for-mommy/oh_daddy.md (hunk 1 above)
Comment (casey-collab) [inline english/cry-out-for-mommy/oh_daddy.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/cry-out-for-mommy/oh_daddy.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Poojap1512** [comment]
"Please review this, @casey-collab".

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**